robot/client/client.go

```diff
@@ -107,6 +107,18 @@ func New(ctx context.Context, address string, logger golog.Logger, opts ...Robot
 	} else {
 		refreshTime = *rOpts.refreshEvery
 	}
+	var checkConnectedTime time.Duration
+	if rOpts.refreshEvery == nil {
+		checkConnectedTime = 10 * time.Second
+	} else {
+		checkConnectedTime = *rOpts.checkConnectedEvery
+	}
+	var reconnectTime time.Duration
+	if rOpts.refreshEvery == nil {
+		reconnectTime = 10 * time.Second
```

Comment:
use the defaults from dialRobotClient

Reply:
Could you elaborate a bit more? I think I may have fixed this.

Reply:
the defaults should be 10 seconds for checkConnectedTime and 1 second for reconnect time

Reply:
So just to confirm 10 for refreshTime and checkConnectedTime. 1 second for reconnectTime?

Reply:
yep

Reply:
cool cool, fixed